divi/src/miner.cpp

```diff
@@ -273,6 +273,22 @@ class BlockMemoryPoolTransactionCollector
             }
         }
     }
+    void ComputeCoinBaseTransaction (
+        CBlock& pblock, 
+        unique_ptr<CBlockTemplate>& pblocktemplate,
+        const bool& fProofOfStake, 
+        const int& nHeight,
+        CMutableTransaction& txNew,
+        const CAmount& nFees)
+    {
+        pblock.vtx[0].vin[0].scriptSig = CScript() << nHeight << OP_0;
+        if (!fProofOfStake) {
+            txNew.vout[0].nValue = GetBlockSubsidity(nHeight).nStakeReward;
+            txNew.vin[0].scriptSig = CScript() << nHeight << OP_0;
+            pblock.vtx[0] = txNew;
+            pblocktemplate->vTxFees[0] = -nFees;
+        }
+    }
```

Comment:
This function extraction is duplicated on line 183-198. No error results because of namespacing